src/main/resources/database/changes/release-22/changelog.yml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+
+- changeSet:
+    id: 22-1
+    author: Jon Fox
+    changes:
+    - sqlFile:
+        comment: Add active_enrolment column to case table
+        path: add-activeenrolment-to-case.sql
```

Comment:
nit picking here - but could we have a dash between active and enrolment for readabilty

Reply:
Done.